README.md

```diff
@@ -9,6 +9,28 @@
 
 ----
 
+### Quickstart
+
+There are multiple ways to install the `backup-restore-operator`, including via the Cluster Explorer UI in Rancher v2.5+, but this example will focus on using `helm` to install the Helm charts.
+Clone the [repository](https://github.com/rancher/backup-restore-operator), `cd` into it, and execute the following commands:
+
+```bash
+helm install --create-namespace -n cattle-resources-system rancher-backup-crd charts/rancher-backup-crd
+helm install -n cattle-resources-system rancher-backup charts/rancher-backup
+```
+
+You can continue to the [User flow](#User-flow) from here, or uninstall the charts.
+If you are uninstalling and want to keep backup(s), ensure that you have created Backup CR(s) and that your backups are stored in a safe location.
```

Comment:
Make uninstall its own header and section

Reply:
Will move it to the dev/contrib file.